akka-http-core/src/main/java/akka/http/javadsl/model/StatusCodes.java

```diff
@@ -57,9 +57,27 @@ private StatusCodes() {}
     public static final StatusCode GONE = akka.http.scaladsl.model.StatusCodes.Gone();
     public static final StatusCode LENGTH_REQUIRED = akka.http.scaladsl.model.StatusCodes.LengthRequired();
     public static final StatusCode PRECONDITION_FAILED = akka.http.scaladsl.model.StatusCodes.PreconditionFailed();
+    public static final StatusCode PAYLOAD_TOO_LARGE = akka.http.scaladsl.model.StatusCodes.PayloadTooLarge();
+
+    /**
+     * @deprecated deprecated in favor of Payload Too Large
```

Comment:
Can you put the exact new symbol name into all of the deprecation messages? I think that would help people following up on that change. So, here it would be `PAYLOAD_TOO_LARGE`:

```suggestion
     * @deprecated deprecated in favor of PAYLOAD_TOO_LARGE
```